### PR TITLE
Added code necessary to support Surround

### PR DESCRIPTION
**Pull request by mgbellemare**
This is adding support for Surround. This requires adding the select button, leading to a little bit of redundancy between the modes PR and this one. When the modes PR is complete, we'll need to remove the starting actions from this game's settings.

### CODE REVIEW

File: src/games/Roms.cpp [hunk 1]
```diff
@@ -185,3 +187,11 @@ RomSettings *buildRomRLWrapper(const std::string &rom) {
     return NULL;
 }
 
+// Taken from Xitari.
```
Comment (mcmachado) [inline src/games/Roms.cpp]: Don't you have to add a comment at the header, like I did for the lives signal?

**Comment by mcmachado** [comment]
I didn't test it, but it LGTM.
I thought you were going to PR to my repository, not this one...

File: src/games/Roms.cpp (hunk 1 above)
Comment (mgbellemare) [inline src/games/Roms.cpp]: Good idea.

**Comment by nikonikolov** [comment]
Was Surround added eventually? I cannot seem to find it in the master branch.

**Pull request closed without merging**